src/test/java/org/dataloader/DataLoaderValueCacheTest.java

```diff
@@ -30,11 +32,12 @@
 
 public class DataLoaderValueCacheTest {
 
-    @Test
-    public void test_by_default_we_have_no_value_caching() {
-        List<List<String>> loadCalls = new ArrayList<>();
+    @ParameterizedTest
+    @MethodSource("org.dataloader.fixtures.parameterized.TestDataLoaderFactories#getWithoutPublisher")
```

Comment:
As mentioned in the description, we use `getWithoutPublisher` rather than `get` as `ValueCache` does not appear to interact well with the `Publisher` `DataLoader`s.